css/global.css

```
@@ -66,3 +67,58 @@ h4 {
   font-size: 13px;
   font-weight: 400;
 }
+
+.button {
+  background-color: #072ac8;
```

Deberiamos usar variables para no hardcodear los colores :)
```suggestion
  background-color: var(--color-primary);
```